packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.spec.ts

```diff
@@ -584,13 +623,50 @@ describe('StripeUPEPaymentStrategy', () => {
                         }
                     });
 
+                    it('fires additional action requires payment method', async () => {
+                        const errorResponse = new RequestError(getResponse({
+                            ...getErrorPaymentResponseBody(),
+                            errors: [
+                                { code: 'additional_action_required' },
+                            ],
+                            additional_action_required: {
+                                type: 'additional_action_requires_payment_method',
+                                data: {
+                                    redirect_url: 'https://redirect-url.com',
+                                    token: 'token'
+                                },
+                            },
+                            status: 'error',
+                        }));
+
+                        jest.spyOn(paymentActionCreator, 'submitPayment')
+                            .mockReturnValue(of(createErrorAction(PaymentActionType.SubmitPaymentFailed, errorResponse)));
+
+                        try {
+                            await strategy.execute(getStripeUPEOrderRequestBodyMock());
+                        } catch (error) {
+                            expect(orderActionCreator.submitOrder).toHaveBeenCalled();
+                            expect(paymentActionCreator.submitPayment).toHaveBeenCalledTimes(2);
+                            expect(stripeUPEJsMock.confirmPayment).toHaveBeenCalledTimes(1);
+                        }
+                    });
+
                     it('throws stripe error if empty payment intent is sent', async () => {
                         const requiredFieldErrorResponse = new RequestError(getResponse({
                             ...getErrorPaymentResponseBody(),
                             errors: [
-                                { code: 'required_field' },
+                                { code: 'additional_action_required' },
                             ],
+                            additional_action_required: {
+                                type: 'additional_action_requires_payment_method',
+                                data: {
+                                    redirect_url: 'https://redirect-url.com',
+                                    token: 'token'
+                                },
+                            },
+                            status: 'error',
                         }));
+
```

Comment:
I think there is no need for an extra line since the previous one is a constant too